plugin/CactbotEventSource/CactbotEventSource.cs

```diff
@@ -379,6 +394,7 @@ protected override void Update()
       // onZoneChangedEvent: Fires when the player changes their current zone.
       string zone_name = Advanced_Combat_Tracker.ActGlobals.oFormActMain.CurrentZone;
       if (notify_state_.zone_name == null || !zone_name.Equals(notify_state_.zone_name)) {
+        fate_watcher_.RemoveAndClearFates();
```

Comment:
onZoneChangedEvent fires _after_ we receive the network FATE data, causing the fates to be added -> zone change -> fate removed -> fate readded the next time an update event is sent.

Reply:
Pushed https://github.com/quisquous/cactbot/pull/903/commits/e30a31f283ee7d55ec495a52514e2ca11c9a41b7 to resolve this.